README.md

```diff
@@ -7,3 +7,6 @@ Repository containing content for the summer of tech source control 101 session.
 Luke Ryan @lukeryannetnz http://lukeryan.net.nz
 
 Max Annear-Henderson
+
```

Comment:
+1 awesome

On Thursday, May 7, 2015, NicholasParry <notifications@github.com> wrote:

> In README.md
> <https://github.com/lukeryanxero/summer-of-tech/pull/58#discussion_r29828308>
> :
>
> > @@ -7,3 +7,6 @@ Repository containing content for the summer of tech source control 101 session.
> >  Luke Ryan @lukeryannetnz http://lukeryan.net.nz
> >
> >  Max Annear-Henderson
> > +
>
> Banana for scale. <http://i.imgur.com/HsT3sC0.jpg>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lukeryanxero/summer-of-tech/pull/58/files#r29828308>.
>


-- 

Luke Ryan

Mobile +64 22 105 6284

Skype luke.ryan.xero


Level 1, 125 St Georges Bay Road, Parnell


Xero - Beautiful accounting software

We're hiring! www.xero.com/careers

www.xero.com